cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -175,7 +175,7 @@ private Timer.Context markAndTime(String subject) {
     }
 
     @OpenApi(
-            description = "Create new TimeSeries, will store any data provided",
+            description = "Used to create and save time-series data. Data to be stored must have time stamps in UTC represented as epoc milliseconds ",
```

Comment:
"epoc" should be "epoch"

Reply:
Thanks!  -- This helps in some documentation review.